### PR TITLE
Add footnote that explicitly mentions that we use nonce as singular

### LINKED ISSUE #45: secnonce/pubnonce naming
This is more a question than a recommendation, but considering that secnonce and pubnonce are technically concatenations of two nonces, which later have to be split up for a LC calculation, wouldn't pluralizing the names to `secnonces` and `pubnonces` make it a bit more intuitive to readers and implementers?

### PR DESCRIPTION
**Pull request by jonasnick**
Attempt to fix #45. CC @arik-so

### CODE REVIEW

File: bip-musig2.mediawiki [hunk 1]
```diff
@@ -67,7 +67,7 @@ When implementing the specification, make sure to understand this section thorou
 
 The basic order of operations to create a multi-signature with the specification is as follows:
 The signers start by exchanging public keys and computing an aggregate public key using the ''KeyAgg'' algorithm.
-When they want to sign a message, each signer starts the signing session by running ''NonceGen'' to compute ''secnonce'' and ''pubnonce''.
+When they want to sign a message, each signer starts the signing session by running ''NonceGen'' to compute ''secnonce'' and ''pubnonce''<ref>We treat the 64-byte ''secnonce'' and 66-byte ''pubnonce'' as grammatically singular despite the fact that they are serializations of two scalars and two elliptic curve points respectively. This treatment may be confusing for readers familiar with with MuSig2 paper. However, serialization is a technical detail that is irrelevant for users of MuSig2 interfaces.</ref>.
```
Comment (real-or-random) [inline bip-musig2.mediawiki]: micro nit:
```suggestion
When they want to sign a message, each signer starts the signing session by running ''NonceGen'' to compute ''secnonce'' and ''pubnonce''<ref>We treat the 64-byte ''secnonce'' and 66-byte ''pubnonce'' as grammatically singular despite the fact that they are serializations of two scalars and two elliptic curve points, respectively. This treatment may be confusing for readers familiar with with MuSig2 paper. However, serialization is a technical detail that is irrelevant for users of MuSig2 interfaces.</ref>.
```

**Review by real-or-random — APPROVED** [review]
ACK

File: bip-musig2.mediawiki (hunk 1 above)
Comment (jonasnick) [reply bip-musig2.mediawiki]: done

Comment (arik-so) [reply bip-musig2.mediawiki]: another nit: replace "with with MuSig2 paper" with "with the MuSig2 paper"

**Review by arik-so — APPROVED** [review]
LGTM!

File: bip-musig2.mediawiki [hunk 2]
```diff
@@ -67,7 +67,7 @@ When implementing the specification, make sure to understand this section thorou
 
 The basic order of operations to create a multi-signature with the specification is as follows:
 The signers start by exchanging public keys and computing an aggregate public key using the ''KeyAgg'' algorithm.
-When they want to sign a message, each signer starts the signing session by running ''NonceGen'' to compute ''secnonce'' and ''pubnonce''.
+When they want to sign a message, each signer starts the signing session by running ''NonceGen'' to compute ''secnonce'' and ''pubnonce''<ref>When they want to sign a message, each signer starts the signing session by running ''NonceGen'' to compute ''secnonce'' and ''pubnonce''<ref>We treat the 64-byte ''secnonce'' and 66-byte ''pubnonce'' as grammatically singular even though they are serializations of two scalars and two elliptic curve points, respectively. This treatment may be confusing for readers familiar with the MuSig2 paper. However, serialization is a technical detail that is irrelevant for users of MuSig2 interfaces.</ref>.
```
Comment (flack) [inline bip-musig2.mediawiki:70]: I think there's a closing `</ref>` missing

Comment (flack) [reply bip-musig2.mediawiki:70]: (or one opening `<ref>` too many)

**Pull request merged**